src/lib.rs

```diff
@@ -97,10 +97,19 @@
 //! To create a custom template, execute these steps:
 //!
 //! 1. Run this command:
```

Comment:
This is such good info to have. One thing that I don't understand is what does `hc scaffold template clone` do? Would it be worth mentioning it here?

Reply:
`hc scaffold template clone` will create a template from the current template you are using, pretty meta but this was how it was originally defined